chat-plugins/chat-monitor.js

```diff
@@ -114,6 +129,17 @@ let chatfilter = function (message, user, room) {
 			return false;
 		}
 	}
+	for (let i = 0; i < filterWords.shorteners.length; i++) {
+		let [regex] = filterWords.shorteners[i];
+		if (typeof regex === 'string') continue; // Failsafe to appease typescript.
+		if (regex.test(lcMessage)) {
+			if ((room && ((room.chatRoomData && room.id.endsWith('staff')) || room.id.startsWith('help-'))) || user.isStaff || (this.pmTarget && this.pmTarget.isStaff)) return `${message} __[shortener: ${String(regex).slice(3, -3).replace('\\.', '.')}]__`;
```

Comment:
```suggestion
			if ((room && ((room.chatRoomData && room.id.endsWith('staff')) || room.id.startsWith('help-'))) || user.isStaff || (this.pmTarget && this.pmTarget.isStaff)) {
				return `${message} __[shortener: ${String(regex).slice(3, -3).replace('\\.', '.')}]__`;
			}
```

Reply:
Honestly I'll need to refactor this anyway; I guess I'll just make it a separate commit. Have fun dealing with the merge conflict.

Reply:
```suggestion
			if (isStaff) return `${message} __[shortener: ${String(regex).slice(3, -3).replace('\\.', '.')}]__`;
```